.eslintrc.js

```diff
@@ -13,49 +13,55 @@ module.exports = {
     "plugin:import/typescript",
     "plugin:prettier/recommended",
   ],
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "allowImportExportEverywhere": true,
-    "ecmaVersion": 6,
-    "sourceType": "module",
-    "project": "./tsconfig.json",
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    allowImportExportEverywhere: true,
+    ecmaVersion: 6,
+    sourceType: "module",
+    project: "./tsconfig.json",
   },
-  "rules": {
+  rules: {
     "babel/generator-star-spacing": 0,
-    
+
     // A function with a name starting with an uppercase letter should only be
     // used as a constructor
-    "babel/new-cap": [1, {
-      "capIsNewExceptions": [
-        "Optional",
-        "OneOf",
-        "Maybe",
-        "MailChimpAPI",
-        "Juice",
-        "Run",
-        "AppComposer",
-        "Query",
-        "Map",
-        "List"
-      ]
-    }],
+    "babel/new-cap": [
+      1,
+      {
+        capIsNewExceptions: [
+          "Optional",
+          "OneOf",
+          "Maybe",
+          "MailChimpAPI",
+          "Juice",
+          "Run",
+          "AppComposer",
+          "Query",
+          "Map",
+          "List",
+        ],
+      },
+    ],
     "babel/array-bracket-spacing": 0,
     "babel/object-curly-spacing": 0,
     "babel/object-shorthand": 0,
     "babel/arrow-parens": 0,
     "eol-last": 1,
     "no-await-in-loop": 0,
     "comma-dangle": 0,
-    "eqeqeq": [1, "always", {"null": "ignore"}],
+    eqeqeq: [1, "always", { null: "ignore" }],
     "key-spacing": 0,
     "no-extra-boolean-cast": 0,
     "no-undef": 1,
-    "no-unused-vars": [1, {
-      "vars": "all",
-      "args": "none",
-      "varsIgnorePattern": "React|PropTypes|Component"
-    }],
-    "no-console": 1,
+    "no-unused-vars": [
+      1,
+      {
+        vars: "all",
+        args: "none",
+        varsIgnorePattern: "React|PropTypes|Component",
+      },
+    ],
+    "no-console": ["warn", { allow: ["info", "warn", "error"] }],
```

Comment:
This is the only change I did to this file, but then the autoformatter kicked in 😅